### DISCUSSION TITLE
Is there a Problem today with public.pool and NerdMiner V2?

### DISCUSSION THREAD

**Discussion by GuentherKnau** [BitMaker-hub/NerdMiner_v2#479] [General]
Today i saw that both of my miners 0.00 Hasrate displaying. So i resetted one of them an reconnected the Miner. It´s still not working.
Has anybody same Issues?

**Comment by adisorin** [comment]
Yes, I have the same problem. I have ESP32-C3 and they all died!

**Comment by adisorin** [comment]
From 53331 to 26085 miners..... a provider was also removed as it had many lost packets. ![2024-09-07-104937_1080x1920_scrot](https://github.com/user-attachments/assets/3c73e068-dfcd-44be-a07f-1add02989424)

**Comment by BitMaker-hub** [comment]
Hello, yes, there is a problem with public pool currently. Public pool is migrating his VPN services

**Reply by adisorin** [reply]
That explains everything. How long does it last?
Miners consume electricity.

**Comment by GuentherKnau** [comment]
Thanks for the Informations!

**Comment by BitMaker-hub** [comment]
I don't know, there are many nerdminers there. You can now move to other pool like nerdminers.org
